eslint.config.mjs

```diff
@@ -0,0 +1,25 @@
+import importPlugin from 'eslint-plugin-import';
+import stylelintConfig from 'eslint-config-stylelint';
+import stylelintJestConfig from 'eslint-config-stylelint/jest';
+
+export default [
+	{
+		ignores: ['**/*.cjs', '.coverage/*'],
```

Comment:
[suggestion] A few `.cjs` files should be listed because they aren't generated by Rollup:

```sh-session
$ git ls '**/*.cjs' | grep -v 'lib'
.changeset/changelog-stylelint.cjs
scripts/visual-config.cjs
system-tests/005/commonjs.test.cjs
system-tests/005/config.cjs
```

```suggestion
		ignores: ['lib/**/*.cjs', '.coverage/*'],
```

See also:

https://github.com/stylelint/stylelint/blob/b6e6a719e0e9924b60ca2b7fc1b989ab3e2f8a07/rollup.config.mjs#L11